libnetdata/socket/security.h

```diff
@@ -33,6 +33,10 @@ extern SSL_CTX *netdata_client_ctx;
 extern SSL_CTX *netdata_srv_ctx;
 extern const char *security_key;
 extern const char *security_cert;
+#if OPENSSL_VERSION_NUMBER >= 0x10100000L
```

Comment:
It seems `0x10100000L` is a magic number, i suggest to fix it

Reply:
This is the way OpenSSL defines their version, this is define inside `/usr/include/openssl/opensslv.h`. What kind of fix are you talking about? Do we need to create a name definition for this version?

Reply:
yes, i meant const or something

Reply:
i see you are using it in 8 places

Reply:
I changed for a text.

Reply:
But I see here you're still using `0x10100000L` instead of `OPENSSL_VERSION_WITH_TLS_13`